core/state/database.go

```diff
@@ -148,68 +170,178 @@ func NewDatabaseWithConfig(db ethdb.Database, config *trie.Config) Database {
 	}
 }
 
+func (db *cachingDB) InTransition() bool {
+	return db.started && !db.ended
+}
+
+func (db *cachingDB) Transitioned() bool {
+	return db.ended
+}
+
+// Fork implements the fork
+func (db *cachingDB) StartVerkleTransition(originalRoot, translatedRoot common.Hash, chainConfig *params.ChainConfig, cancunBlock *big.Int) {
+	fmt.Println(`
+	__________.__                       .__                .__                   __       .__                               .__          ____         
+	\__    ___|  |__   ____        ____ |  |   ____ ______ |  |__ _____    _____/  |_     |  |__ _____    ______    __  _  _|__| ____   / ___\ ______
+	  |    |  |  |  \_/ __ \     _/ __ \|  | _/ __ \\____ \|  |  \\__  \  /    \   __\    |  |  \\__  \  /  ___/    \ \/ \/ |  |/    \ / /_/  /  ___/
+	  |    |  |   Y  \  ___/     \  ___/|  |_\  ___/|  |_> |   Y  \/ __ \|   |  |  |      |   Y  \/ __ \_\___ \      \     /|  |   |  \\___  /\___ \
+	  |____|  |___|  /\___        \___  |____/\___  |   __/|___|  (____  |___|  |__|      |___|  (____  /_____/       \/\_/ |__|___|  /_____//_____/
+                                                    |__|`)
+	db.started = true
+	db.AddTranslation(originalRoot, translatedRoot)
+	db.baseRoot = originalRoot
+	// initialize so that the first storage-less accounts are processed
+	db.StorageProcessed = true
+	chainConfig.CancunBlock = cancunBlock
```

Comment:
Nit: maybe rename `cancunBlock` since won't be cancun.

Reply:
that's the midterm plan, but there's a caveat with this: geth won't let prague be activated without cancun, I have to see if this is going to be a problem. While we only shoot for shappella, it's better to keep it as a fake cancun.